src/server/db_slice.cc

```diff
@@ -330,9 +330,9 @@ void DbSlice::Reserve(DbIndex db_ind, size_t key_size) {
   db->prime.Reserve(key_size);
 }
 
-auto DbSlice::Find(const Context& cntx, string_view key, unsigned req_obj_type) const
-    -> OpResult<PrimeIterator> {
-  auto it = FindInternal(cntx, key, FindInternalMode::kDontUpdateCacheStats).first;
+OpResult<PrimeIterator> DbSlice::Find(const Context& cntx, string_view key,
```

Comment:
why we dont have pre update in this flow?

Reply:
`Find()` is the last function to be converted. Once we merge this PR, the only API that will be using it is `FindFirst()`, which I will work on in a next PR.